blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,286 @@
+# common
+common.date.duration.format=d[d] h[h] m[m] s[s]
+common.date.duration.hint.format=M [month], d [days], h[ hours], m[ minutes], s[ seconds]
+common.date.duration.display.format=d[d] h[h] m[m] s[s]
+common.date.readable.long=MMM DD YYYY h:mma Z
+common.date.readable.short=MMM DD h:mma Z
+#加载中...
+common.pager.loading=\u52a0\u8f7d\u4e2d\u002e\u002e\u002e
```

Comment:
Well the full clone is slow here, I downloaded the raw properties file directly from here[1] instead. It is still the same when opened in Vim and Xcode. Tried to open it in Poedit, but it doesn't support the file format. I checked the other languages files, French, German and Russian, they are all the same. All the non-ascii characters will be displayed in unicode.

[1] https://raw.githubusercontent.com/meiboyu/blueocean-plugin/c248695190cd24bb23805e0724dfe10441a343bc/blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties